src/EnergyPlus/SolarShading.cc

```diff
@@ -771,7 +771,7 @@ void AllocateModuleArrays(EnergyPlusData &state)
     state.dataHeatBal->CosIncAngHR.dimension(24, state.dataSurface->TotSurfaces, 0.0);
     state.dataHeatBal->CosIncAng.dimension(state.dataGlobal->NumOfTimeStepInHour, 24, state.dataSurface->TotSurfaces, 0.0);
     state.dataHeatBal->SurfAnisoSkyMult.dimension(state.dataSurface->TotSurfaces,
```

Comment:
FWIW I would also like us to stop using `dimension` and replace with `allocate` and explicit loop initialization. This will allow us to combine the initializations multiple arrays into a single loop, which improves performance.